cloudbridge/cloud/providers/gce/resources.py

```diff
@@ -1311,11 +1311,26 @@ def name(self, value):
 
     @property
     def description(self):
-        return self._volume.get('description')
+        self.refresh()
```

Comment:
We had a bit of a discussion on this particular `refresh` and concluded that unless there is a technical necessity to keep it, it'd be better to remove it as a way to keep the response time and the number of requests down. We're going to try and standardize on that across the library and make it explicit that the user code should call the object's `refresh` method rather than making a decision when the library ought to do it.